mono/metadata/threadpool-worker-default.c

```diff
@@ -221,6 +221,7 @@ rand_next (gpointer *handle, guint32 min, guint32 max)
 static void
 destroy (gpointer data)
 {
+#if 0
```

Comment:
I am going to squash the commit, so the PR message is going to be part of the commit message. And I don't want to ammend the commit, as it's going to retrigger the whole build, which will take 4 more hours, and my understanding is that it's pretty time sensitive, to get the ios team some builds and testing.